plugin/bundleVisualizer.ts

```diff
@@ -65,10 +65,23 @@ async function bundleVisualizer(this: ServerlessAnalyzeBundlePlugin): Promise<vo
 
   await pExec(`unzip .serverless/${functionZipName} -d ${TEMP_DIR_LOCATION}`);
 
-  const test = getAllFiles(`${TEMP_DIR_LOCATION}`);
-  const metafileName = test.filter(
-    fileName => fileName.includes(`/${functionName}/`) && fileName.endsWith('-meta.json'),
-  )[0];
+  const allFiles = getAllFiles(`${TEMP_DIR_LOCATION}`);
+  const metafileName = allFiles.filter(fileName => {
+    const handlerPath = (slsFunction as FunctionDefinitionHandler).handler.split('.')[0];
+
+    return fileName.includes(handlerPath) && fileName.endsWith('-meta.json');
+  })[0];
+
+  if (!metafileName) {
+    this.serverless.cli.log(
+      `🤯 Analyze failed: function ${functionName} metadata was not found`,
+      'ServerlessAnalyzeBundlePlugin',
+      { color: 'red' },
+    );
+
+    return;
+  }
```

Comment:
Add explicit error handler here, since I was getting uncaught errors prior to fixing this.